### PR TITLE
V2 - Composer controlled WP Core version

### PR DESCRIPTION
**Pull request by ksere**
PR opened for the purpose of reviewing. **NOT TO BE MERGED**

Rel. [task](https://app.clickup.com/t/862kjndjn)

### CODE REVIEW

**Review by dpanta94 — APPROVED** [review]
LGTM!

File: handle-wp-core-update.sh [hunk 1]
```diff
@@ -67,4 +67,4 @@ fi
 
 # If versions don't match, update.
 echo "Updating WordPress Core to version $CORE_VERSION_COMPOSER_TO."
-run_command "wp core update --version=$CORE_VERSION_COMPOSER_TO --force --path=${REMOTE_ROOT}"
+run_command "cd ${REMOTE_ROOT} && wp core update --version=$CORE_VERSION_COMPOSER_TO --force"
```
Comment (dpanta94) [inline handle-wp-core-update.sh:81]: @ksere I'll test it to be sure but wouldn't the `cd` performed on L39 make this `cd` fail ?

**Review by dpanta94 — COMMENTED** [review]
Just a comment.

File: handle-wp-core-update.sh (hunk 1 above)
Comment (dpanta94) [reply handle-wp-core-update.sh:81]: Performed review on outdated commit, possible issue is still there though but on different Line

File: prepare-wp-core-update.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+cd "${GITHUB_WORKSPACE}/${PATH_DIR}" || exit 1;
+
+cp "$GITHUB_ACTION_PATH/handle-wp-core-update.sh" "${RUNNER_TEMP}/handle-wp-core-update.sh"
+
+CORE_VERSION_COMPOSER_TO=$(composer config extra.wordpress-core) || CORE_VERSION_COMPOSER_TO=""
+
+cd "${GITHUB_WORKSPACE}/${FROM_DIR}" || exit 1;
+
+CORE_VERSION_COMPOSER_FROM=$(composer config extra.wordpress-core) || CORE_VERSION_COMPOSER_FROM=""
```
Comment (dpanta94) [inline prepare-wp-core-update.sh:17]: Issue here during onboarding. That would never be set for the first time the extras.wordpress-core is being added to a composer.json.

File: handle-wp-core-update.sh (hunk 1 above)
Comment (ksere) [reply handle-wp-core-update.sh:81]: The cd here is to an absolute path

File: prepare-wp-core-update.sh (hunk 2 above)
Comment (ksere) [reply prepare-wp-core-update.sh:17]: yes, but it should not stop the thing from running

**Pull request closed without merging**